sources/openmetrics/openmetrics.go

```diff
@@ -2,35 +2,156 @@ package openmetrics
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"io"
 	"net/http"
+	"net/url"
 	"regexp"
+	"sort"
+	"time"
 
 	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
 	"github.com/sirupsen/logrus"
+	"github.com/stripe/veneur/v14"
+	"github.com/stripe/veneur/v14/samplers"
+	"github.com/stripe/veneur/v14/sources"
+	"github.com/stripe/veneur/v14/util"
 )
 
-type PrometheusResults struct {
+type OpenMetricsSourceConfig struct {
+	HistogramBucketTag string        `yaml:"histogram_bucket_tag"`
+	Host               util.Url      `yaml:"host"`
+	IgnoredMetrics     util.Regexp   `yaml:"ignored_metrics"`
```

Comment:
Ignored metric names only - @arnavdugar-stripe mentioned we want to replace this with the more advanced regex routing rules

Reply:
Added docs in the README.md file clarifying this.